Adding scoping services volumes and IT configs

EdwinFLopez opened the pull request:
Updating to include scoping service data exporters extensions in the docker-dev volumes section and some config required to run integration tests in local dev envs.

EdwinFLopez commented:
@rossbrandon for your consideration. Please let me know of any change you would like me to include here. Thanks.

rossbrandon commented on README.md:
```diff
@@ -245,9 +245,11 @@ Otherwise, this project now automatically sets up Xdebug support with VS Code. I
         - `<path_to_project>/extensions/commerce-data-export-ee/ProductOverrideDataExporter` -> `/var/www/html/app/code/Magento/ProductOverrideDataExporter`
         - `<path_to_project>/extensions/saas-export/SaaSCatalog` -> `/var/www/html/app/code/Magento/SaaSCatalog`
         - `<path_to_project>/extensions/saas-exportSaaSCommon` -> `/var/www/html/app/code/Magento/SaaSCommon`
+        - `<path_to_project>/extensions/saas-exportSaaSScopes` -> `/var/www/html/app/code/Magento/SaaSScopes`
```
Looks like this and the above line are missing a `/` -> `<path_to_project>/extensions/saas-export/SaaSScopes`

EdwinFLopez replied:
nice catch....

EdwinFLopez replied:
fixed.

rossbrandon commented on compose/docker-compose.dev.yml:
```diff
@@ -51,3 +53,19 @@ services:
     volumes: *appvolumes
+
+  ## Adding Elastic Search to run integration tests in local dev environment
+  elastic:
```
Should we comment this out by default and describe in documentation that it should be uncommented is using integration tests? This would help save docker resource usage.

EdwinFLopez replied:
Agree.

EdwinFLopez replied:
done.

rossbrandon commented on compose/env/db.env:
```diff
@@ -0,0 +1,11 @@
+# File required to configure and run integration tests in local env.
```
To reduce confusion since the main db variables are in `env/install.env` can we rename this file `integration_test_db.env` or something like that?

EdwinFLopez replied:
Unfortunately, it needs to be named like this for the integration tests to get setup. It is being called by the php setup tests scripts, and otherwise, the setup will fail. I'll look further into this.

rossbrandon replied:
Ah, I see. Can we put it in a `env/tests` directory?

EdwinFLopez replied:
It is referenced by `bin/setup-integration-tests.`. We can update such script to reference `env/install.env` and remove it, or remove the db config section from the `env/install.env` and add `source env/db.env` right after `source env/install.env` to `bin/setup`. What would you prefer?

rossbrandon replied:
Oof. That sounds messy, haha. Probably easier to keep it as it is.

rossbrandon reviewed (changes requested):
Thanks for doing this! I just have a few small changes requested.

EdwinFLopez commented:
@rossbrandon : I've resolved your comments and updated one of the scripts to update the phpunit.xml.dist to the container in case it has changes. wdyt? Also, updated the README.md to add the integration tests section.
